remote/advancedcontainers/set-docker-compose-project-name.md

```diff
@@ -24,8 +26,10 @@ $env:COMPOSE_PROJECT_NAME=foo
 code .
 ```
 
-Or add the following to a `.env` file in the root of the project (**not** in the `.devcontainer` folder):
+Or add the following to a `.env` file in the same folder as the `docker-compose.yml`:
 
 ```
 COMPOSE_PROJECT_NAME=foo
 ```
+
+When no project name is configured and the `docker-compose.yml` is in the `.devcontainer` folder the Docker Compose default of using the `docker-compose.yml` folder's basename is overridden with `${project-folder-basename}_devcontainer` to avoid name collisions with other projects.
```

Comment:
```suggestion
When no project name is configured and the `docker-compose.yml` is in the `.devcontainer` folder, the Docker Compose default of using the `docker-compose.yml` folder's basename is overridden with `${project-folder-basename}_devcontainer` to avoid name collisions with other projects.
```